src/services/claim/MerkleOrchardV1Config.json

```diff
@@ -0,0 +1,73 @@
+{
+  "1": [
+    {
+      "label": "NEXO",
+      "distributor": "0x0000000000000000000000000000000000000000",
+      "token": "0xB62132e35a6c13ee1EE0f84dC5d40bad8d815206",
+      "manifest": "https://raw.githubusercontent.com/balancer/bal-mining-scripts/master/reports/_current-nexo.json",
+      "weekStart": 1
+    },
+    {
+      "label": "BANK",
+      "distributor": "0xc38c5f97B34E175FFd35407fc91a937300E33860",
+      "token": "0x2d94AA3e47d9D5024503Ca8491fcE9A2fB4DA198",
+      "manifest": "https://raw.githubusercontent.com/balancer/bal-mining-scripts/master/reports/_current-bankless.json",
+      "weekStart": 12
+    }
+  ],
+  "42": [
+    {
+      "label": "BAL",
+      "distributor": "0x95FaE1C936B4Cd6c5099d7A705D792ee6aC9FEc3",
+      "token": "0x41286Bb1D3E870f3F750eB7E1C25d7E48c8A1Ac7",
+      "manifest": "https://raw.githubusercontent.com/balancer/bal-mining-scripts/sample-tree/reports-kovan/_current.json",
+      "weekStart": 52
+    },
+    {
+      "label": "DAI",
+      "distributor": "0x95FaE1C936B4Cd6c5099d7A705D792ee6aC9FEc3",
+      "token": "0x04DF6e4121c27713ED22341E7c7Df330F56f289B",
+      "manifest": "https://raw.githubusercontent.com/balancer/bal-mining-scripts/sample-tree/reports-kovan/_current-dai.json",
+      "weekStart": 1
+    }
+  ],
+  "137": [
+    {
+      "label": "WMATIC",
+      "distributor": "0xBd44C01EC7d623372B4572247afB6231eDD8486F",
+      "token": "0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270",
+      "manifest": "https://raw.githubusercontent.com/balancer/bal-mining-scripts/master/reports/_current-wmatic-polygon.json",
+      "weekStart": 4
+    },
+    {
+      "label": "WMATIC",
+      "distributor": "0x632208491602Dd205da8Cb9C0BA98620fc19854A",
+      "token": "0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270",
+      "manifest": "https://raw.githubusercontent.com/balancer/bal-mining-scripts/master/reports/_current-wmatic-polygon.json",
+      "weekStart": 5
+    },
+    {
```

Comment:
Also this entire TUSD block can go - there were 2 distributors and both were recovered and redistributed in a single one